### ISSUE TITLE
ffi_standalone_check_number_1.0.7

### ISSUE THREAD

**Issue by kmavrommatis** [tidyverse/ggplot2#5235]
Hi,

trying to create a simple ggplot figure I encounter the error
**Error in .standalone_types_check_dot_call(ffi_standalone_check_number_1.0.7,  : 
  object 'ffi_standalone_check_number_1.0.7' not found**

R version 4.2.2 (2022-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Monterey 12.6.1
ggplot: 3.4.1

Here is the code to reproduce the bug (same applies with other geometries and more complex data):

```r
test=cbind(x=c(1,2,4),y=c(2,4,4))

test %>% as.data.frame() %>% ggplot(aes(x=x,y=y)) + geom_point()
Error in .standalone_types_check_dot_call(ffi_standalone_check_number_1.0.7,  : 
  object 'ffi_standalone_check_number_1.0.7' not found
```

Thanks in advance for your help

**Comment by teunbrand** [comment]
This should currently only occur if you use the development version of ggplot2. If you update the {rlang} package to 1.1.0, everything should (in theory) work as expected. Could I ask you to let us know if updating the rlang package has resolved the problem?

**Comment by kmavrommatis** [comment]
Thanks for the quick response. Indeed updating rlang to 1.0.0 solved the issue.

I am still unclear for the reason this discrepancy occured, I didn't (conciously) install `ggplo2`. I installed ggplot2 through `install.packages( "ggplot2" )` as usual.

Thanks again for your help.

**Comment by teunbrand** [comment]
I agree that this is strange. What does `packageVersion("ggplot2")` return?

**Comment by 178627** [comment]
> I agree that this is strange. What does `packageVersion("ggplot2")` return?

![image](https://user-images.githubusercontent.com/128213052/226092866-5c9b7d93-3fad-490b-9cd6-9b5e6bd67c78.png)

**Comment by teunbrand** [comment]
@178627 The error should resolve if you update rlang to 1.1.0, but I'm still puzzled why 3.4.1. is trying to use the new number check. What does `rlang::last_trace()` tell once you've triggered the error?

**Comment by thomasp85** [comment]
It might be from gtable which was just updated with the new rlang type checking. However it has a version dependency on rlang 1.1.0 so I don't know how you'd get to a state where rlang 1.1.0 wasn't installed

**Comment by brianmsm** [comment]
Same issue here. In my case it was because I had errors when updating rlang in windows (folder permissions that could not be copied in the installation). I had to uninstall rlang and manually delete the folder for the new installation to work without errors.

**Comment by lnarwhale** [comment]
> ```r
> http://cran.us.r-project.org
> ```

Thank you for your reply, I try it, but it's still a Error
<img width="591" alt="image" src="https://user-images.githubusercontent.com/83813393/226553595-3f967091-92df-45e9-8228-04ca8921dfd4.png">

**Comment by Krishnakumarshrestha** [comment]
got this error when i install gtable from cran . previously i had similar error like in the discussion above . then i install gtable .now error is like this

Error in draw_axis(break_positions = guide$key[[aesthetic]], break_labels = guide$key$.label,  : 
  lazy-load database 'C:/Users/DELL/AppData/Local/R/win-library/4.2/gtable/R/gtable.rdb' is corrupt
In addition: Warning message:
In draw_axis(break_positions = guide$key[[aesthetic]], break_labels = guide$key$.label,  :
  internal error -3 in R_decompress1

**Comment by Krishnakumarshrestha** [comment]
<img width="649" alt="Screenshot 2023-03-25 085901" src="https://user-images.githubusercontent.com/45503377/227689018-2068f180-4ab7-4244-b5d4-83f4e5173f39.png">

**Comment by thomasp85** [comment]
Please try to restart your r session after installing the package

**Comment by ab55468346** [comment]
真厉害

**Comment by teunbrand** [comment]
The recently released version of ggplot2 3.4.2 have fixed this, I think.
